omod/src/main/webapp/resources/scripts/conditionlist/controllers/editcondition.controller.js

```diff
@@ -92,7 +95,12 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
         var groups = document.getElementsByClassName("group");
         if($scope.condition.status == INACTIVE_STATUS){
             groups[2].style.visibility = "visible";
-            inputs[3].value = CommonFunctions.formatDate($scope.condition.endDate); 
+
+            if($scope.condition.endDate==null){
```

Comment:
```suggestion
            if ($scope.condition.endDate == null) {
```

Always put spaces around operators (except for named function parameters in languages that have them).